src/biogeophys/SoilStateInitTimeConstMod.F90

```diff
@@ -765,12 +765,14 @@ real(r8) function ThresholdSoilMoistZender2003( clay )
   !------------------------------------------------------------------------------
       real(r8), intent(IN) :: clay ! Fraction of clay in the soil (%)
 
+      real(r8), parameter :: a = 0.5_r8   ! Tuning factor for the soil moisture effect. This parameter could also be added into ThresholdSoilMoistKok2014. Simone Tilmes suggested to change this parameter into a namelist variable for easier CESM tuning. dmleung 30 Sep 2024.
```

Comment:
@dmleung note since this function is used by both Leung and Zender, it changes answers for both.

So I'm going to modify the code slightly so that it will be identical for Zender, by passing in the "a" parameter with it hardcoded to 1 for Zender. Let me know if you have other thoughts on that.

Reply:
Good point Erik, I forgot about this. I could also do it the other way, but it's good that you can also take care of it. At least I will also change this in my next commit.